[
  {
    "signature": "SubscriberExceptionContext(com.google.common.eventbus.EventBus eventBus,java.lang.Object event,java.lang.Object subscriber,java.lang.reflect.Method subscriberMethod)",
    "name": "SubscriberExceptionContext",
    "containingClass": {
      "qualifiedName": "com.google.common.eventbus.SubscriberExceptionContext",
      "name": "SubscriberExceptionContext",
      "isArray": false
    },
    "targetClass": "com.google.common.eventbus.SubscriberExceptionContext",
    "isVarArgs": false,
    "parameters": [
      {
        "type": {
          "qualifiedName": "com.google.common.eventbus.EventBus",
          "name": "EventBus",
          "isArray": false
        },
        "name": "eventBus"
      },
      {
        "type": {
          "qualifiedName": "java.lang.Object",
          "name": "Object",
          "isArray": false
        },
        "name": "event"
      },
      {
        "type": {
          "qualifiedName": "java.lang.Object",
          "name": "Object",
          "isArray": false
        },
        "name": "subscriber"
      },
      {
        "type": {
          "qualifiedName": "java.lang.reflect.Method",
          "name": "Method",
          "isArray": false
        },
        "name": "subscriberMethod"
      }
    ],
    "paramTags": [
      {
        "parameter": {
          "type": {
            "qualifiedName": "com.google.common.eventbus.EventBus",
            "name": "EventBus",
            "isArray": false
          },
          "name": "eventBus"
        },
        "comment": "The EventBus that handled the event and the subscriber. Useful for broadcasting a a new event based on the error.",
        "kind": "PARAM",
        "condition": ""
      },
      {
        "parameter": {
          "type": {
            "qualifiedName": "java.lang.Object",
            "name": "Object",
            "isArray": false
          },
          "name": "event"
        },
        "comment": "The event object that caused the subscriber to throw.",
        "kind": "PARAM",
        "condition": ""
      },
      {
        "parameter": {
          "type": {
            "qualifiedName": "java.lang.Object",
            "name": "Object",
            "isArray": false
          },
          "name": "subscriber"
        },
        "comment": "The source subscriber context.",
        "kind": "PARAM",
        "condition": ""
      },
      {
        "parameter": {
          "type": {
            "qualifiedName": "java.lang.reflect.Method",
            "name": "Method",
            "isArray": false
          },
          "name": "subscriberMethod"
        },
        "comment": "the subscribed method.",
        "kind": "PARAM",
        "condition": ""
      }
    ],
    "throwsTags": []
  },
  {
    "signature": "getEventBus()",
    "name": "getEventBus",
    "containingClass": {
      "qualifiedName": "com.google.common.eventbus.SubscriberExceptionContext",
      "name": "SubscriberExceptionContext",
      "isArray": false
    },
    "targetClass": "com.google.common.eventbus.SubscriberExceptionContext",
    "isVarArgs": false,
    "returnType": {
      "qualifiedName": "com.google.common.eventbus.EventBus",
      "name": "EventBus",
      "isArray": false
    },
    "parameters": [],
    "paramTags": [],
    "returnTag": {
      "comment": "The EventBus that handled the event and the subscriber. Useful for broadcasting a a new event based on the error.",
      "kind": "RETURN",
      "condition": ""
    },
    "throwsTags": []
  },
  {
    "signature": "getEvent()",
    "name": "getEvent",
    "containingClass": {
      "qualifiedName": "com.google.common.eventbus.SubscriberExceptionContext",
      "name": "SubscriberExceptionContext",
      "isArray": false
    },
    "targetClass": "com.google.common.eventbus.SubscriberExceptionContext",
    "isVarArgs": false,
    "returnType": {
      "qualifiedName": "java.lang.Object",
      "name": "Object",
      "isArray": false
    },
    "parameters": [],
    "paramTags": [],
    "returnTag": {
      "comment": "The event object that caused the subscriber to throw.",
      "kind": "RETURN",
      "condition": ""
    },
    "throwsTags": []
  },
  {
    "signature": "getSubscriber()",
    "name": "getSubscriber",
    "containingClass": {
      "qualifiedName": "com.google.common.eventbus.SubscriberExceptionContext",
      "name": "SubscriberExceptionContext",
      "isArray": false
    },
    "targetClass": "com.google.common.eventbus.SubscriberExceptionContext",
    "isVarArgs": false,
    "returnType": {
      "qualifiedName": "java.lang.Object",
      "name": "Object",
      "isArray": false
    },
    "parameters": [],
    "paramTags": [],
    "returnTag": {
      "comment": "The object context that the subscriber was called on.",
      "kind": "RETURN",
      "condition": ""
    },
    "throwsTags": []
  },
  {
    "signature": "getSubscriberMethod()",
    "name": "getSubscriberMethod",
    "containingClass": {
      "qualifiedName": "com.google.common.eventbus.SubscriberExceptionContext",
      "name": "SubscriberExceptionContext",
      "isArray": false
    },
    "targetClass": "com.google.common.eventbus.SubscriberExceptionContext",
    "isVarArgs": false,
    "returnType": {
      "qualifiedName": "java.lang.reflect.Method",
      "name": "Method",
      "isArray": false
    },
    "parameters": [],
    "paramTags": [],
    "returnTag": {
      "comment": "The subscribed method that threw the exception.",
      "kind": "RETURN",
      "condition": ""
    },
    "throwsTags": []
  }
]